### PR TITLE
ICDS UI changes

### PR DESCRIPTION
**Pull request by kkrampa**
@calellowitz Could you merge https://github.com/dimagi/commcare-hq/pull/17674 before this one?

### CODE REVIEW

File: custom/icds_reports/utils.py [hunk 1]
```diff
@@ -243,6 +243,30 @@ def apply_exclude(domain, queryset):
     )
 
 
+def get_age_filter(age_value):
+    """
+        When age_value comes from age filter it means first range is chosen 0-6 months.
+        For that range we want to include 0 and 6 in results.
+    """
+    if age_value == '6':
```
Comment (calellowitz) [inline custom/icds_reports/utils.py]: can you explain this part a little more, im not sure i understand what is going on here

**Review by calellowitz — CHANGES_REQUESTED** [review]
one question then it looks good to go

File: custom/icds_reports/utils.py [hunk 2]
```diff
@@ -1414,6 +1476,9 @@ def get_data_for(filters, date):
             weighed=Sum('wer_weighed'),
             all=Sum('wer_eligible'),
         )
+        if show_test:
```
Comment (emord) [inline custom/icds_reports/utils.py]: I think this should be `not show_test` (there are some others that need to be switched as well)

**Review by emord — CHANGES_REQUESTED** [review]

File: custom/icds_reports/utils.py (hunk 2 above)
Comment (kkrampa) [reply custom/icds_reports/utils.py]: Fixed in all places

File: custom/icds_reports/utils.py (hunk 1 above)
Comment (kkrampa) [reply custom/icds_reports/utils.py]: We have a bunch of values in filter
0 - 6 months, 6 - 12 months, etc.
currently, age_tranche can have a values 0, 6, 12, 18 etc.
To not lose value '0' we need to include it in the first range 0-6. For the rest of ranges, we include only rightmost value.

0 -6 -> 0,6
6-12 -> 12
12-18 -> 18
etc.
As you can see without this clause we lose 0 value at all.

**Comment by kkrampa** [comment]
@emord @calellowitz 
Updated

**Review by emord — APPROVED** [review]

**Comment by emord** [comment]
This PR now has a file conflict

**Comment by kkrampa** [comment]
@calellowitz @emord Updated

**Review by calellowitz — APPROVED** [review]

**Pull request merged**